src/streamlink/plugins/twitch.py

```diff
@@ -421,18 +423,30 @@ def access_token(self, is_live, channel_or_vod):
             validate.union_get("signature", "value"),
         )
 
-        return self.call(query, schema=validate.Schema(
-            {"data": validate.any(
+        return self.call(query, acceptable_status=(200, 401), schema=validate.Schema(
```

Comment:
Are these the only status codes that are ever returned from a "successful" request?

Reply:
400 and 403 could maybe be added, but apart from that, I don't think anything else will be useful when trying to parse the `{"error": str, ...}` response schema. The remaining 4xx errors or any 5xx errors will be handled by the `HTTPSession` where it'll raise a `PluginError` instead.

Reply:
Sounds good to me. Figured it was worth double checking just in case so there's no issue when this goes live that will require a second release 👍